R/target_capacity.R

```diff
@@ -1,18 +1,28 @@
-#' @title Title
+#' @title Target Capacity
 #'
 #' @description
-#' A short description...
+#' Calculates the target capacity to achieve a given target waiting time as a 
+#' function of observed demand, target waiting time and a variability of demand 
+#' parameter F.
+#' 
+#' F defaults to 1.
+#' 
+#' Target Capacity = Demand + 2 * ( 1 + 4 * F ) / Target Wait
 #'
 #' @param demand
```

Comment:
@PeterSNHS would you be able to add a one-liner for each of the @params?

Reply:
Added parameter descriptions in 91d7d22